notes.txt

```diff
@@ -0,0 +1,16 @@
+When all todos are deleted render a button to add new todo
+
+Make it pretty.
+
+
+Redis for ordering lists?
+
+Create a login screen
```

Comment:
Perhaps remove this from our code :). You can add notes.txt to our gitignore if you'd like.